### PR TITLE
ci: configure workflow for pull request title checker

### PR DESCRIPTION
**Pull request by Neeraj-swarnkar**
# COMPLETES #[SPARK-385629](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-385629)

Configured workflow for pull request title checker

< DESCRIBE THE CONTEXT OF THE ISSUE >

< DESCRIBE YOUR CHANGES >

Configured workflow for pull request title checker
Added below 2 files - 
.github/pr-title-checker-config.json
.github/workflows/pr-title-checker.yml

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor
- [x] Changes to our CI configuration files and scripts

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: .github/workflows/pr-title-checker.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: webex/pr-title-checker
```
Comment (Neeraj-swarnkar) [inline .github/workflows/pr-title-checker.yml:15]: https://github.com/webex/pr-title-checker
Forked in Webex and checking if workflow is trigerring or not.

File: .github/pr-title-checker-config.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "LABEL": {
+      "name": "title needs formatting",
+      "color": "EEEEEE"
+    },
+    "CHECKS": {
+      "prefixes": ["build", "chore", "ci", "docs", "feat", "fix", "perf", "refactor", "revert", "test"],
```
Comment (Neeraj-swarnkar) [inline .github/pr-title-checker-config.json:7]: PR Title should have these prefixes

**Comment by Neeraj-swarnkar** [comment]
@arun3528, can you merge this, I will raise new PR to check if it's triggering workflow or not.

File: .github/pr-title-checker-config.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+	"LABEL": {
+		"name": "title needs formatting",
+		"color": "EEEEEE"
+	},
+	"CHECKS": {
+		"prefixes": ["build: ", "chore: ", "ci: ", "docs: ", "feat: ", "fix: ", "perf: ", "refactor: ", "revert: ", "test: "],
```
Comment (Neeraj-swarnkar) [inline .github/pr-title-checker-config.json:7]: PR Title should have these prefixes and regex, which is working fine in Embedded app SDK.

File: .github/workflows/pr-title-checker.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      - labeled
+      - unlabeled
+    branches:
+      - master
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR title
+        uses: webex/pr-title-checker@main
```
Comment (Neeraj-swarnkar) [inline .github/workflows/pr-title-checker.yml:20]: https://github.com/webex/pr-title-checker
This Action is forked in Webex, we checked this workflow is trigerring in Embedded app SDK.

https://github.com/webex/embedded-app-sdk/actions/workflows/pr-title-checker.yml

**Comment by Neeraj-swarnkar** [comment]
@arun3528, Please check. Thanks.

**Review by arun3528 — APPROVED** [review]

**Pull request merged**